doc/tools/docgram/dune

```diff
@@ -0,0 +1,10 @@
+(executable
+ (name doc_grammar)
+ (libraries coq.clib coqpp))
+
+(rule
+ (targets fullGrammar)
+ (deps ../../../parsing/g_constr.mlg ../../../parsing/g_prim.mlg ../../../toplevel/g_toplevel.mlg ../../../vernac/g_proofs.mlg ../../../vernac/g_vernac.mlg ../../../plugins/btauto/g_btauto.mlg ../../../plugins/cc/g_congruence.mlg ../../../plugins/derive/g_derive.mlg ../../../plugins/extraction/g_extraction.mlg ../../../plugins/firstorder/g_ground.mlg ../../../plugins/funind/g_indfun.mlg ../../../plugins/ltac/coretactics.mlg ../../../plugins/ltac/extraargs.mlg ../../../plugins/ltac/extratactics.mlg ../../../plugins/ltac/g_auto.mlg ../../../plugins/ltac/g_class.mlg ../../../plugins/ltac/g_eqdecide.mlg ../../../plugins/ltac/g_ltac.mlg ../../../plugins/ltac/g_obligations.mlg ../../../plugins/ltac/g_rewrite.mlg ../../../plugins/ltac/g_tactic.mlg ../../../plugins/ltac/profile_ltac_tactics.mlg ../../../plugins/micromega/g_micromega.mlg ../../../plugins/micromega/g_zify.mlg ../../../plugins/nsatz/g_nsatz.mlg ../../../plugins/omega/g_omega.mlg ../../../plugins/rtauto/g_rtauto.mlg ../../../plugins/setoid_ring/g_newring.mlg ../../../plugins/syntax/g_numeral.mlg ../../../plugins/syntax/g_string.mlg)
```

Comment:
Maybe you want to use a `glob` here?

Reply:
That shortens things a bit, thanks! But the glob language is too limited to make things as short as in `Makefile.doc`.

Reply:
I was under the impression that `**.mlg` did work in Dune.

Reply:
It exists, but it doesn't do what you think. See https://dune.readthedocs.io/en/stable/concepts.html#glob, and in particular:

>Currently dune only supports globbing files in a single directory.

Reply:
See also ocaml/dune#1866

Reply:
Oh indeed.